lib/github.js

```diff
@@ -5,7 +5,6 @@ var spawn = require('child_process').spawn
 var github = exports;
 
 function Organization(options) {
-    this.BASE_URI = 'https://api.github.com/orgs/';
```

Comment:
Your solution is probably more simple. Although I have an aversion to any logic living in constructors. It's just a force of habit to make things more testable.

Having said that, I'll make the change and resubmit.

Cheers!